services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -692,8 +699,12 @@ class ApplicationServiceITest : IntegrationTest() {
 
         @Test
         fun `Skips the area inside hanke check with a generated hanke`() {
```

Comment:
Had to read this couple of times before figuring it out. Maybe just my bad wiring but would it help also others if the name was something like `Skips the area-inside-hanke check with a generated hanke()`?

Reply:
I renamed the test.